orienteer-camel/src/main/java/org/orienteer/camel/component/OIntegrationConfig.java

```diff
@@ -0,0 +1,168 @@
+package org.orienteer.camel.component;
+
+import java.io.ByteArrayInputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.ServiceStatus;
+import org.apache.camel.impl.DefaultCamelContext;
+import org.apache.camel.model.RouteDefinition;
+import org.apache.camel.model.RoutesDefinition;
+import org.apache.wicket.Application;
+import org.apache.wicket.Component;
+import org.apache.wicket.authorization.UnauthorizedActionException;
+import org.orienteer.camel.behavior.OIntegrationConfigStopBehavior;
+import org.orienteer.camel.tasks.CamelEventHandler;
+import org.orienteer.camel.tasks.OCamelTaskSessionCallback;
+import org.orienteer.camel.widget.CamelWidget;
+import org.orienteer.core.component.BootstrapType;
+import org.orienteer.core.component.FAIconType;
+import org.orienteer.core.method.ClassOMethod;
+import org.orienteer.core.method.IMethodEnvironmentData;
+import org.orienteer.core.method.OFilter;
+import org.orienteer.core.method.filters.PlaceFilter;
+import com.orientechnologies.orient.core.record.impl.ODocument;
+import com.orientechnologies.orient.core.type.ODocumentWrapper;
+
+import ru.ydn.wicket.wicketorientdb.IOrientDbSettings;
+import ru.ydn.wicket.wicketorientdb.OrientDbWebApplication;
+import ru.ydn.wicket.wicketorientdb.OrientDbWebSession;
+
+/**
+ * Wrapper for OIntegrationConfig ODocuments
+ *
+ */
+
+public class OIntegrationConfig extends ODocumentWrapper {
+	private static final long serialVersionUID = 1L;
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////	
+	@ClassOMethod(
+			order=10,bootstrap=BootstrapType.SUCCESS,icon = FAIconType.play,
+			filters={
+					@OFilter(fClass = PlaceFilter.class, fData = "STRUCTURE_TABLE"),
+//					@OFilter(fClass = PlaceFilter.class, fData = "STRUCTURE_TABLE|DATA_TABLE"),
+			}
+	)
+	public void start(IMethodEnvironmentData data){
+		final CamelContext context = getOrMakeContextByRid(getDocument().getIdentity().toString(),data.getCurrentWidget());
+		new Thread(new Runnable() {
+			@Override
+			public void run() {
+				try {
+					if (context.getStatus().isSuspended()){
+						context.resume();
+						//target.add(CamelWidget.this.form);
+					}else if (!context.getStatus().isStarted()){
+						clearContext(context);
+						String script = getDocument().field("script");
+						RoutesDefinition routes = context.loadRoutesDefinition(new ByteArrayInputStream( script.getBytes()));
+						context.addRouteDefinitions(routes.getRoutes());
+						context.start();
+					}
+				} catch (Exception e) {
+					// TODO Auto-generated catch block
+					e.printStackTrace();
+				}
+			}
+		}).start();
+	}
+/////////////////////////////////////////////////////////////////////////////////////////////////////	
+	
+	@ClassOMethod(
+			order = 30,bootstrap=BootstrapType.DANGER,icon = FAIconType.stop,
```

Comment:
Now it autogeneratedas "OClass.method".It is for more accurate identification and avoid doubles,but i can add title key